### DISCUSSION TITLE
Print Stylesheet

### DISCUSSION THREAD

**Discussion by mikevalstar** [withastro/starlight#1490] [Feature Requests]
### What version of `starlight` are you using?

0.18.0

### What is your idea?

An `@media print` based stylesheet to support printable documentation.

### Why is this feature necessary?

This is useful when sharing portions of documentation with people who cannot access the published webpage and allows the resulting PDF to be searchable and sharable. 

I would be happy to contribute this feature.

### Participation

- [X] I am willing to submit a pull request for this feature.

**Comment by delucis** [comment]
Thanks for the post @mikevalstar!

In case anyone else ends up here, we’re tracking this in https://github.com/withastro/starlight/discussions/1160
